bin/docraptor

```diff
@@ -53,6 +53,14 @@ opt_parser = OptionParser.new do |opts|
     options[:async] = async
   end
 
+  opts.on("--callback_url url", "Set a callback URL to be notified upon success. Defaults to naught") do |callback_url|
+    # callback_url can be used for local testing. You need three consoles:
+    #   nc -lk 9001                                                       # listen on port 9001 for any data
+    #   ngrok http 9001                                                   # setup external URL; port from nc session
+    #   bin/docraptor content --callback_url "http://1111111111.ngrok.io" # copied url from the ngrok session
```

Comment:
Mostly I think `examples/` is where we put these in the other client repos, but any of the solutions can work.